File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageElementString.java
package com.dhsdevelopments.potato.clientapi.message;

public class MessageElementString extends MessageElement
{
    private String value;

    public MessageElementString( String value ) {
        this.value = value;
    }

    @Override
    public CharSequence getSpannable() {
        return value;
    }

    @Override
    public String toString() {
        return "MessageElementString[" +
                       "value='" + value + '\'' +
                       "] " + super.toString();
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageElementString.kt
package com.dhsdevelopments.potato.clientapi.message

class MessageElementString(private val value: String) : MessageElement() {

    override fun makeSpan(): CharSequence {
        return value
    }

    override fun toString(): String {
        return "MessageElementString[" +
                "value='" + value + '\'' +
                "] " + super.toString()
    }
}
